app.py

```diff
@@ -0,0 +1,139 @@
+import streamlit as st
+from PIL import Image
+import altair as alt
+
+#this is what allows for the option menu along the sidebar 
+from streamlit_option_menu import option_menu
+
+#until nesta_ds_utils is hosted on pypi we won't be able to use it for apps hosted on streamlit cloud
+#once that's the case, we should replace this with nesta_ds_utils altair formatting function
+from utils.formatting import *
+alt.themes.register("nestafont", nestafont)
+alt.themes.enable("nestafont")
+colours = NESTA_COLOURS
+
+#INSERT THE TITLE OF YOUR APP HERE
+APP_TITLE = 'SAMPLE STREAMLIT APP'
+
+
+#icon to be used as the favicon on the browser tab
+im = Image.open("images/favicon.ico")
+
+# sets page configuration with favicon and title specified on line 4
+st.set_page_config(
+    page_title=APP_TITLE, 
+    layout="wide", 
+    page_icon=im
+)
+
+def app(app_title: str):
+    """
+    THIS IS A TEMPLATE APP TO GET YOU STARTED WITH BUILDING STREAMLIT APPS
+    """
+    header = st.container()
+    with header:
+        #nesta logo
+        nesta_logo = Image.open(f"images/nesta_logo.png")
+
+        #set title of app to be title specified
+        st.title(app_title)
+        col1, col2 = st.columns([1, 10])
+        with col1:
+            # add nesta logo on left side 
+            st.image(nesta_logo, width=100)
+        with col2:
+            # add nesta website next to logo
+            st.markdown(
+                "           **website:** https://www.nesta.org/"
+            )
+        #Indicate that this dashboard is still a draft. Remove once final.
+        st.markdown(
+            "**This dashboard is a under active development and plots and metrics should not be interpreted as final**"
+        )
+    
+    #this creates a sidebar on the left of the app with the option menu    
+    with st.sidebar:
+        choose = option_menu(
+        app_title,
+        ["PAGE_TITLE1", "PAGE_TITLE2", # replace with your desired page title names
+        ],
+        icons=["bookmark-star", "bookmark-star" #bookmarks can be any icons available from https://icons.getbootstrap.com/
+        ],
+        default_index=0,
+        orientation="vertical",
+        styles={
+            "container": {
+                "padding": "5!important",
```

Comment:
Out of curiosity, what does 5!important do?

Reply:
No idea.. i copied it from the iod app :) will remove it and see if it's actually needed

Reply:
Ahhah yep that checks out, let me know, I copied it from an example! I did it because it allowed for those customisable icons, whereas streamlit multipage only seemed to allow classic emojis (that you couldn't change the colour of). I also felt like it looked nicer (but that's just my personal opinion!). But it is likely easier to just have multipage and then people can add that if they want it!

Reply:
I have looked it up. It is CSS syntax, I have removed it as it didn't seem to make a difference.